android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -1127,15 +1137,19 @@ public void setResizeModeModifier(@ResizeMode.Mode int resizeMode) {
     }
 
     private void applyModifiers() {
+        Log.d("ReactExoplayerView", "applyModifiers() - repeat=" + repeat + ", muted=" + muted + ", volume=" + audioVolume);
         setRepeatModifier(repeat);
         setMutedModifier(muted);
+        setVolumeModifier(audioVolume);
```

Comment:
This will set the audioVolume to whatever was provided as a prop to the component. The default value is 1.0, and this initialization wasn't happening, which is the source of the bug.